.hemtt/lints.toml

```diff
@@ -7,3 +7,17 @@ options.ignore = [
 options.ignore = [
     "SLX_*"
 ]
+
+[sqf.undefined]
+enabled = true
+options.check_orphan_code = false
+
+[sqf.unused]
+enabled = false
+options.check_params = false
+
+[sqf.shadowed]
+enabled = false
+
+[sqf.not_private]
+enabled = false
```

Comment:
I don't want these committed in case they change or never see main HEMTT, that was an issue in the past with random things getting copy pasted around